src/Draco.FormatterEngine/Draco.FormatterEngine.csproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
I mean it's your project here but generally for non-tests we keep this off

Reply:
The VS template. I'll fix that.